calcom/cal.com
# Google Meet - installable app

## What does this PR do?



This app makes Google Meet an installable app.

Fixes # (issue)



https://www.loom.com/share/04c50b7fdbfd403780859f4dc9277fcd

**Environment**: Staging(main branch) / Production

## Type of change




## How should this be tested?



- Go into the app store and install Google Meet
- You do need a Google calendar connected, so connect one if not already done so
- If you do not have a calendar, then it will automatically take you to connect one
- Change an event type's location to Meet. You should see a warning
- Book a meeting, the Meet link should appear in the email
- Change the destination calendar to a non Google one. It should fallback to Cal Video

## Checklist


- I haven't added tests that prove my fix is effective or that my feature works
- I haven't checked if new and existing unit tests pass locally with my changes
